nmdc_automation/workflow_automation/watch_nmdc.py

```diff
@@ -292,7 +319,7 @@ def update_operation(self, opid, done, meta):
 class Watcher:
     """ Watcher class for monitoring and managing jobs """
     def __init__(self, site_configuration_file: Union[str, Path],  state_file: Union[str, Path] = None):
-        self._POLL = 20
+        self._POLL = 60
```

Comment:
Could you clarify what this interval is (seconds I presume) in the comments?

Reply:
Renamed prop